fdbclient/GlobalConfig.actor.cpp

```diff
@@ -44,6 +44,32 @@ GlobalConfig& GlobalConfig::globalConfig() {
 	ASSERT(res);
 	return *reinterpret_cast<GlobalConfig*>(res);
 }
+void GlobalConfig::applyChanges(Transaction& tr,
+                                const Standalone<VectorRef<KeyValueRef>>& insertions,
+                                const Standalone<VectorRef<KeyRangeRef>>& clears) {
```

Comment:
I think `insertions` and `clears` here could be taken as non-standalone vector refs. It doesn't change anything with our current usages, but it would avoid a copy if anybody ever called this with a non-standalone.